src/ansys/fluent/core/session.py

```diff
@@ -103,15 +106,24 @@ def __init__(self, fluent_connection: FluentConnection):
         Args:
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
         """
-        BaseSession.build_from_fluent_connection(self, fluent_connection)
+        remote_file_handler = RemoteFileHandler(
+            transfer_service=PimFileTransferService(fluent_connection._remote_instance)
```

Comment:
As commented previously, any dependency on a file transfer services will be passed in. Follow-up work, no action required.

Reply:
It's moved to `__init__` of `BaseSession`.